app/views/journeys/expenses/travelAndAccommodation/AddAnotherVehicleView.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.common.{BusinessId, TaxYear, UserType}
+@import controllers.journeys.expenses.travelAndAccommodation.routes
+@import views.html.components.Heading
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    heading: Heading,
+    govukSummaryList: GovukSummaryList,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode, vehicleName: SummaryList, userType: UserType, taxYear: TaxYear, businessId: BusinessId)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages(s"addAnotherVehicle.title.$userType"))) {
+
+    @formHelper(action = routes.AddAnotherVehicleController.onSubmit(taxYear, businessId, mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
```

Comment:
There's a component you can use for this called ErrorSummarySection, which does all the heavy lifting for you. Then, you can just write

`@errorSummarySection(form)`

You'll need to inject it first (add it to the `@this()` list)